templates/corporate/remote_billing_finalize_login_confirmation.html

```diff
@@ -22,6 +22,19 @@ <h1>Log in to Zulip plan management for<br/>{{ host }}</h1>
             {% endif %}
         </div>
         <div class="white-box">
+            {% if show_server_on_live_plan_error or show_any_realm_in_server_on_live_plan_error %}
+            <div id="remote-billing-confirm-login-error" class="alert alert-danger">
+                {% if show_any_realm_in_server_on_live_plan_error %}
+                Plan management for this server is not available because at least one organization hosted on
+                this server already has an active plan. Please contact <a href='mailto:{{ support_email }}'>contact support</a> with any questions.
+                {% else %}
+                Plan management is not available for this organization,
+                because your Zulip <b>server is already on a {{ server_plan_name }} plan</b>, which covers all
```

Comment:
Is the bold section supposed to just be the plan name?

Reply:
I actually wanted to highlight what is the real reason among all the text.